src/main.rs

```diff
@@ -625,7 +619,7 @@ fn bisect(cfg: &Config, client: &Client) -> Result<(), Error> {
 
 fn searched_range(
     cfg: &Config,
-    searched_toolchains: &Vec<Toolchain>,
+    searched_toolchains: &[Toolchain],
```

Comment:
warning: writing `&Vec<_>` instead of `&[_]` involves one more reference and cannot be used with non-Vec-based slices.